### PR TITLE
STAR Interview Technique

### LINKED ISSUE #262: Week 1 PD

### PR DESCRIPTION
**Pull request by zoefarrell**

### CODE REVIEW

File: module4/lessons/Week1/STARInterviewTechnique.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+layout: page
+title: STAR Interview Technique
+---
+
+## Learning Goals
+* Understand how to use the STAR technique when answering interview questions
+* Begin to gather a collection of stories to use in a behavioral interview
+
+### Preparation
+Before this session, review the three strengths you identified in the "Your Strengths" PD session.
```
Comment (memcmahon) [inline module4/lessons/Week1/STARInterviewTechnique.md:11]: Should we add a calendar event for this so it doesn't fall off our radar?

File: module4/lessons/Week1/STARInterviewTechnique.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+### Warmup
```
Comment (memcmahon) [inline module4/lessons/Week1/STARInterviewTechnique.md:13]: Love that this warmup puts the students in the interviewer's seat!

File: module4/lessons/Week1/STARInterviewTechnique.md [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+
+When you are not answering the question, listen closely to the person who is speaking so you can share feedback:
+* Did they include all four parts of the STAR technique?
+* Did they speak in specifics (ex: “I did XYZ”) or just in hypotheticals (ex: “I would do XYZ”)?
+* Did they provide enough context to help you understand their story?
+
+Once you have both had a chance to answer, give feedback and discuss how you felt that went.
+
+<aside class="instructor-notes" markdown="1">
+<p><strong>Instructor Note</strong><br>Because this is the student's first time trying STAR I would give them 2 minutes to think about what story they want to tell before putting them in breakout rooms.</p>
```
Comment (memcmahon) [inline module4/lessons/Week1/STARInterviewTechnique.md]: I think this is a great call-out.  We could even make this part of the activity outline.

**Review by memcmahon — APPROVED** [review]
This looks great!

File: module4/lessons/Week1/STARInterviewTechnique.md (hunk 1 above)
Comment (zoefarrell) [reply module4/lessons/Week1/STARInterviewTechnique.md:11]: Good call will add a 10 minute PD prep to the calendar

File: module4/lessons/Week1/STARInterviewTechnique.md (hunk 2 above)
Comment (zoefarrell) [reply module4/lessons/Week1/STARInterviewTechnique.md:13]: Agree! Pulled this question from a FE/BE PD, really like the putting them in the other seat idea.

File: module4/lessons/Week1/STARInterviewTechnique.md (hunk 3 above)
Comment (zoefarrell) [reply module4/lessons/Week1/STARInterviewTechnique.md]: I like that idea!

**Pull request merged**